flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
Have you tested other arches or that's just to be safe?

Reply:
This is here because the build process requires some x86-64 specific code when building on my machine.
I know that it's at least compatible with 32-bit Intel and 32-bit ARM, but unfortunately it does not look like java plays too well with qemu emulation, so I was not able to trigger that process locally on my laptop and adjust the manifest.
I want to extend this to ARM once/if it gets accepted.